grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json

```diff
@@ -3982,14 +3982,14 @@
         "allValue": null,
         "current": {},
         "datasource": "${DS_PROMETHEUS}",
-        "definition": "label_values(chi_clickhouse_metric_Uptime, exported_namespace)",
+        "definition": "label_values({__name__ =~ \"chi_clickhouse_metric_Uptime|chi_clickhouse_metric_fetch_errors\"}, exported_namespace)",
```

Comment:
fix for corner case when clickhouse cluster is offline (wrong configuration, for example)